python-client/giskard/ml_worker/testing/tests/performance.py

```diff
@@ -596,8 +596,8 @@ def test_diff_rmse(actual_slice: Dataset, reference_slice: Dataset, model: Model
           Slice of the actual dataset
         reference_slice(Dataset):
           Slice of the actual dataset
-        model(Model):
-          Model used to compute the test
+        model(BaseModel):
+          BaseModel used to compute the test
```

Comment:
everywhere in docstrings in this file "Model used" referred just to a "model" word rather than to a "Model" class

Reply:
I think it can be kept "Model" as it's a new line, so a non-biased user who doesn't know about our initial naming of `Model` the class, wouldn't consider it as a class.
```            
            actual_slice(Dataset):
              Slice of the actual dataset
            model(BaseModel):
              Model used to compute the test
            threshold(float):
              Threshold value for Accuracy
 ```